src/main/java/com/fasterxml/jackson/databind/ser/DefaultSerializerProvider.java

```diff
@@ -617,9 +617,6 @@ protected Impl(SerializerProvider src, SerializationConfig config,
         @Override
         public DefaultSerializerProvider copy()
         {
-            if (getClass() != Impl.class) {
```

Comment:
This is actually necessary -- will revert this change; super impl will throw exception and the intent is to fail if sub-classing provider impl but not overriding `copy()` method.

Reply:
Ohhhh. I see; `Impl` is `final` so it can never occur. Never mind. :)